RadarSDK/RadarReplayBuffer.m

```diff
@@ -150,14 +158,25 @@ - (void)removeReplaysFromBuffer:(NSArray<RadarReplay *> *)replays {
     [mutableReplayBuffer removeObjectsInArray:replays];
 
     // persist the updated buffer
-    NSData *replaysData = [NSKeyedArchiver archivedDataWithRootObject:mutableReplayBuffer];
+    NSData *replaysData;
+    if (@available(iOS 11.0, *)) {
+        replaysData = [NSKeyedArchiver archivedDataWithRootObject:mutableReplayBuffer requiringSecureCoding:true error:nil];
+    } else {
+        replaysData = [NSKeyedArchiver archivedDataWithRootObject:mutableReplayBuffer];
+    }
     [[NSUserDefaults standardUserDefaults] setObject:replaysData forKey:@"radar-replays"];
 }
 
 - (void)loadReplaysFromPersistentStore {
     NSData *replaysData = [[NSUserDefaults standardUserDefaults] objectForKey:@"radar-replays"];
     if (replaysData) {
-        NSArray *replays = [NSKeyedUnarchiver unarchiveObjectWithData:replaysData];
+        NSArray *replays;
+        if (@available(iOS 11.0, *)) {
```

Comment:
I'm a little confused about iOS 11 version gate since NSSecureCoding is available from iOS 6

Reply:
the `archivedDataWithRootObject: requireingSecureCoding:` function is only introduced in iOS 11.0. the function that we were using (without the second param) is the one raising security alerts.